notebooks/dataproc_mimic_III.ipynb

```diff
@@ -343,15 +397,24 @@
     },
     {
       "cell_type": "code",
-      "execution_count": null,
+      "execution_count": 23,
       "metadata": {
         "colab": {
           "base_uri": "https://localhost:8080/"
         },
         "id": "w4tDxreQ6Ysd",
         "outputId": "b2bfe18b-da7e-4708-b068-549c8f0b2c78"
       },
-      "outputs": [],
+      "outputs": [
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "/Users/chrispebble/opt/anaconda3/envs/py37/lib/python3.7/site-packages/IPython/core/interactiveshell.py:2878: DtypeWarning: Columns (4) have mixed types.Specify dtype option on import or set low_memory=False.\n",
```

Comment:
Consider clearing all output on the notebook before saving, for two reasons:
* References to your local machine's file system, as above, are not portable
* On the off chance something in here is considered sensitive

Reply:
Excellent point!  I was afraid to lose the output but I just saved it separately for now.